CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing
+
+We're excited to see you're interested in contributing to Vapor's Redis integration package! This document covers how you can report any issues you find or contribute with bug fixes and new features!
+
+> **Note:** Starting with version 4.0, much of the implementation will be handled by [NIORedis]() and [RedisKit]() - so please consider contributing to those repositories as well!
+
+## Reporting Issues
+
+Go ahead and [open a new issue](https://github.com/vapor/redis/issues/new). The team will be notified and we should get back to you shortly.
+
+We give you a few sections to fill out to help us hunt down the issue more effectively. Be sure to fill everything out to help us get everything fixed up as fast as possible.
+
+## Pull Requests
+
+We are always willing to have pull requests open for bug fixes or new features.
+
+Bug fixes are pretty straight forward. Open up a PR and a maintainer will help guide you to make sure everything is in good order.
+
+New features are little more complex. If you come up with an idea, you should follow these steps:
+
+- [Open a new issue](https://github.com/vapor/redis/issues/new) or post in the #ideas or #development channel on the [Discord server](http://vapor.team/). This lets the us know what you have in mind so we can discuss it from a higher level down to the implementation details.
+- We'll take a look at your proposed idea and decide whether it would be better as a feature for NIORedis, RedisKit, or maybe it isn't needed enough to be added (don't take it personally if we reject it 😄, we still think you're a nice person). If we decide it fits with this package, you can go ahead and open a PR!
```

Comment:
Sorry, but is the part in the parentheses really necessary?

Reply:
The
> (don't take it personally if we reject it 😄, we still think you're a nice person)
part?

This is the file template I've been using that I believe orignated from [NIOKit](https://github.com/vapor/nio-kit/blob/master/.github/CONTRIBUTING.md).

After searching, I only found reference to this in NIOKit and RedisKit (again, because I copied from NIOKit).

Reply:
I wrote that 😄. You can get rid of that for all I care. I guess it comes from all the time I spend on StackOverflow and also all the comments that people make about SO being the worst place ever because people aren't feeling. I've gathered that the problem isn't what the person wrote, but the tone that it is read in, so I try to work a more inclusive feel into my writing.